openy_af4_vue_app/src/App.vue

```diff
@@ -537,6 +542,9 @@ export default {
     },
     showHomeBranchBlock() {
       return !this.hideHomeBranchBlock && this.homeBranchId
+    },
+    getResultsBarClasses() {
```

Comment:
I don't think "get..." is a good name for the computed property. Would advise to remove "get" from here and other computed property names. Just "resultsBarClasses".

Reply:
Fixed.